igvc_gazebo/nodes/ground_truth/ground_truth_republisher.cpp

```diff
@@ -54,8 +77,9 @@ void groundTruthCallback(const nav_msgs::Odometry::ConstPtr& msg)
     // publish odom message
     g_ground_truth_pub.publish(result);
 
-    // publish transform for tf if there has not been a update from the localization node in the last second
-    // since it also publishes the same transform
+    // publish transform for tf if there has not been a update from the
+    // localization node in the last second since it also publishes the same
+    // transform
     if (std::abs(msg->header.stamp.toSec() - g_last_estimate.toSec()) > 1.0)
     {
       static tf::TransformBroadcaster br;
```

Comment:
Maybe instead of having this as a static variable we could move this to the header file?